amundsen_application/static/js/interfaces/Notifications.ts

```diff
@@ -0,0 +1,15 @@
+// TODO: Remove notification types that can be triggered in flask layer if necessary
+export enum NotificationType {
+  ADDED = 'amundsen/notification/OWNER_ADDED',
```

Comment:
I think we should still use simpler strings here. This is following the naming patterns of action enums (which is necessary to make sure we don't have any collisions, as every action needs a unique string), and I'd rather not start mixing in naming patterns associated with one area of code, into another area of code. These could easily be:
```
export enum NotificationType {
  ADDED = 'added',
  REMOVED = 'removed',
  EDITED = 'edited',
  REQUESTED = 'requested',
}
```

Reply:
gotcha, will do